tests/e2e/support/commands.js

```diff
@@ -21,5 +21,5 @@
 // Cypress.Commands.add("dismiss", { prevSubject: 'optional'}, (subject, options) => { ... })
 //
 //
-// -- This will overwrite an existing command --
+// -- This is will overwrite an existing command --
```

Comment:
@ifaouibadi what's that? :D

Reply:
those are custom commands for cypress, they're more like reusable testing chunks.
for ex on each flow, you want to make sure it's connected to the wallet first, we can create a custom command that checks the wallet and reuse it

Reply:
I mean the sentence, why was `is` added? doesn't make sense, does it? 😅

Reply:
ah lol, that's when I regenerated the test files with vue cli

Reply:
really? that's generated? ok, then let's keep it ... 🤦‍♂️